### PR TITLE
A11y

### PR DESCRIPTION
**Pull request by hizo**
https://github.com/nearform/react-pwa/issues/81

Implements:
- skip links
- refactors some titles to use H1, H2 etc to create better document outline
- refactors navigation hamburger, close buttons, previos / next buttons to be actual buttons with proper labels, so we get a keyboard focus on them
- properly implements the side navigation (aria-hidden on main content when it's visible, focus trap on the navigation so you cant tab out of the navigation)

How to test:
- When the app loads, try pressing TAB key and observe skip links to be shown and the focus changing naturally e.g. not selecting hidden navigation elements like before.
- Try also mobile viewport and going to the navigation by keyboard.
- If you have `webdeveloper` toolbar extension installed in chrome, try viewint the document outline from the Information panel

### CODE REVIEW

File: src/client/js/components/More/More.jsx [hunk 1]
```diff
@@ -75,21 +75,25 @@ export function More({ location, data }) {
...
-
-      {nextLinkEnabled && (
-        <Link className={styles.moreItem} to={nextLink}>
-          {'>'}
-        </Link>
-      )}
-
-      {!nextLinkEnabled && <span className={classes(styles.moreItem, styles.moreDisabled)}>{'>'}</span>}
+      <Link
+        className={nextLinkEnabled ? styles.moreItem : classes(styles.moreItem, styles.moreDisabled)}
```
Comment (simoneb) [inline src/client/js/components/More/More.jsx]: you can use the `classes` helper alone to conditionally create a class name. it ignores falsy values. would make this expression a little nicer without the conditional operator

File: src/server/page.html.jsx [hunk 2]
```diff
@@ -121,7 +129,7 @@ export async function renderPage(request, reply) {
           media="(min-device-width: 1024px) and (max-device-width: 1024px) and (-webkit-min-device-pixel-ratio: 2) and (orientation: portrait)"
         />
 
-        <style>{globalStyles.getStyles()}</style>
+        <style dangerouslySetInnerHTML={{ __html: globalStyles.getStyles() }} />
```
Comment (simoneb) [inline src/server/page.html.jsx]: why this change?

Comment (hizo) [reply src/server/page.html.jsx]: because of the general rule I added - `[tabindex="-1"]`. I didnt wan't the quotation marks to be escaped.

Comment (simoneb) [reply src/server/page.html.jsx]: feels like a hammer solution though. I'm confident there must be a way to write the rule that doesn't require this.
